### ISSUE TITLE
Accessing the built source

### ISSUE THREAD

**Issue by muffinresearch** [braintree/card-validator#2]
Consumers of this package probably only want the built source. Would it be possible to make available the built source in npm and bower?

**Comment by muffinresearch** [comment]
I built the source running `npm install` followed by `gulp build` however I needed to use the standalone option e.g passing an options object to `browserify()` with the following:

```javascript
return browserify({
  standalone: 'card-validator',
  entries: config.src.js.main
})
// etc
```

To get a UMD wrapped module that would work with the rest of the AMD setup I'm using:

> When opts.standalone is a non-empty string, a standalone module is created with that name and a umd wrapper. You can use namespaces in the standalone global export using a . in the string name as a separator. For example: 'A.B.C

https://www.npmjs.com/package/browserify#var-b-browserify-files-or-opts

**Comment by jeffcarp** [comment]
Hey @muffinresearch, thank you for bringing this up. Any feedback is very appreciated!

For making it a standalone build - it looks like we can do that without any disruption to existing integrations. I can make a separate issue for that to get signoff and track its progress.

For including a built file in the repo - I don't know of any way around this if we choose to support Bower. My preference is to bite the bullet and include the built file.

@kyledetella, @EvanHahn, @mrak, what are your opinions on this?

**Comment by mrak** [comment]
We are planning on having UMD bundles uploaded with each tagged release. See #6

**Comment by mrak** [comment]
You can download the UMD bundle for the current release at https://github.com/braintree/card-validator/releases/tag/1.0.0 under __Downloads__.

Hope this helps!

**Comment by muffinresearch** [comment]
@mrak how can I get the umd bundle in a package via npm if it's not in the dist dir of the package? If I need to manually download a release this seems to lose a lot of the benefits of having a package. (We aren't using browserify).

It should be possible to have the dist files in the package without necessarily committing to to the tree (although it would seem alot of people do that anyway) see use cases for npm scripts (prepublish specifically) here: https://docs.npmjs.com/misc/scripts

**Comment by muffinresearch** [comment]
@jeffcarp @mrak I note version 2.0.0 doesn't have built files as part of the release here: https://github.com/braintree/card-validator/releases/tag/2.0.0

**Comment by mrak** [comment]
PR #6 has been updated to include `dist` with the npm module and the repository source. Would it work for the build environment you are using?

**Comment by mrak** [comment]
@muffinresearch ^^

**Comment by muffinresearch** [comment]
@mrak cool, lgtm - thanks!

**Comment by mrak** [comment]
@muffinresearch published as `card-validator@2.0.1` let us know if it works for you!

**Comment by muffinresearch** [comment]
Looks great. Thanks for the fix.
